packages/x-components/src/x-modules/facets-next/entities/editable-number-range-filter.entity.ts

```diff
@@ -0,0 +1,55 @@
+import {
+  EditableNumberRangeFilter,
+  Filter,
+  isEditableNumberRangeFilter
+} from '@empathyco/x-types-next';
+import { Store } from 'vuex';
+import { EquatableFilter } from './equatable-filter';
+import { FilterEntity } from './types';
+
+/**
+ * Allows selecting and deselecting a filter of {@link EditableNumberRangeFilter}.
```

Comment:
We should decide wether we document the implementation or not in the entities. Currently `SimpleFilterEntity` and `RawFilterEntity` ([in PR](https://github.com/empathyco/x/pull/73)) are not documented but this is.

I suggest to remove these docs despite each implementation is different. At the end  each entity is "selecting and deselecting" on its own.

Reply:
As we've discussed during the stand up, the rest of entities must include this docs in order to explain how they do the select / deselect process.